lib/instagram_graph_api/client.rb

```diff
@@ -8,8 +8,5 @@ class Client < Koala::Facebook::API
     include InstagramGraphApi::Client::Media
     include InstagramGraphApi::Client::Discovery
 
-    def initialize(media_id = nil, access_token)
-      super(access_token)
-    end
```

Comment:
It didn't look like this custom initializer was ever actually used to removed it so it just defaults to the Koala initializer.